app/src/main/java/com/cornellappdev/transit/models/Route.kt

```diff
@@ -26,7 +26,7 @@ data class Direction(
     @Json(name = "delay") val delay: Int?,
     @Json(name = "startLocation") val startLocation: LatLng,
     @Json(name = "path") val path: List<LatLng>,
-    @Json(name = "type") val type: String,
+    @Json(name = "type") val type: String, // "walk" or "depart"
```

Comment:
These are the two possible values?

I'd recommend making this a `private val type: String`, and then making the actual value an enum of either WALK or DEPART as an extension value (remember the `get() = ...` syntax?) This feels python-y, lol.